bsp/esp32_p4_function_ev_board/examples/wifi_connection/main/idf_component.yml

```diff
@@ -0,0 +1,8 @@
+dependencies:
+  espressif/esp32_p4_function_ev_board:
+    version: "*"
+    override_path: "../../../"
+  espressif/esp_wifi_remote:
+    version: "*"
```

Comment:
Probably makes sense to pin at least some version of esp_wifi_remote (and maybe esp_hosted?) here.

Reply:
I agree, do you plan update and set some major version here?